sdk/test-utils/recorder/GUIDELINES.md

```diff
@@ -237,7 +237,7 @@ Add `@azure/test-utils-recorder` as a devDependency of your sdk.
 - We leverage mocha's `.skip()` functionality to skip the test
   `this.skip()` - https://mochajs.org/#inclusive-tests.
 
-- `{recorder.skip(runtime?: "node" | "browser")}` will skip the test in node or browser runtimes based on the `{runtime}` argument. If the `{runtime}` is undefined, the test will be skipped in both the node and browser runtimes. Has no effect if the `TEST_MODE` is neither `"record"` nor `"playback"`.
+- `{recorder.skip(runtime?: "node" | "browser", reason?: string)}` will skip the test in node or browser runtimes based on the `{runtime}` argument. If the `{runtime}` is undefined, the test will be skipped in both the node and browser runtimes. Has no effect if the `TEST_MODE` is `"record"` or `"live"`.
```

Comment:
so if the mode is "record", skip runs the test, but doesn't record (e.g. it's the same as "live") right?

Reply:
Great catch!! 
I'm sorry, should have been more attentive.

> If the mode is "record", the test will be skipped. Has no effect only in "live".

Fixed here - 9ba9ad6